sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/v2ResolutionPlans.scala

```diff
@@ -42,14 +42,23 @@ case class UnresolvedTable(multipartIdentifier: Seq[String]) extends LeafNode {
 }
 
 /**
- * Holds the name of a table or view that has yet to be looked up in a catalog. It will
- * be resolved to [[ResolvedTable]] or [[ResolvedView]] during analysis.
+ * Holds the name of a table or a permanent/temporary view that has yet to be looked up in
+ * a catalog. It will be resolved to [[ResolvedTable]] or [[ResolvedView]] during analysis.
  */
 case class UnresolvedTableOrView(multipartIdentifier: Seq[String]) extends LeafNode {
```

Comment:
shall we simply add an `acceptTempView` flag here instead of adding `UnresolvedTableOrPermanentView`?